lib/elixir/test/elixir/calendar/holocene.exs

```diff
@@ -78,50 +79,81 @@ defmodule Calendar.Holocene do
     )
   end
 
+  defp zero_pad(val, count) when val >= 0 do
+    num = Integer.to_string(val)
```

Comment:
I think you can use `String.pad_leading/3` to pad.

Reply:
Done, thanks for the comment.